front/app/containers/Admin/projects/components/ProjectRow/index.tsx

```diff
@@ -83,22 +82,18 @@ const ProjectRow = ({
   const { data: authUser } = useAuthUser();
   const projectId = publication.relationships.publication.data.id;
   const publicationStatus = publication.attributes.publication_status;
+  const { data: project } = useProjectById(projectId);
 
   const { data: parentPublication } = useAdminPublication(
     publication.relationships.parent?.data?.id || null
   );
   const localize = useLocalize();
 
-  if (isNilOrError(authUser)) {
+  if (!authUser || !project) {
     return null;
   }
-  const userCanModerateProject =
-    // This means project is in a folder
-    (typeof folderId === 'string' &&
-      userModeratesFolder(authUser.data, folderId)) ||
-    canModerateProject(publication.relationships.publication.data.id, {
-      data: authUser.data,
-    });
+
+  const userCanModerateProject = canModerateProject(project.data, authUser);
```

Comment:
We shouldn't change this because of the way our system works atm. You can technically remove a folder moderator from moderating a project but they should still have access to this project as long as they are a folder moderator. So I think we should leave this as is.

Reply:
Yes, this makes sense. I feel our system is confusing in general because once you assign someone as a folder moderator, you automatically also add them as project moderator to all the projects in that folder. But then, these roles can be modified independently which creates inconsistent logic at times. In any case - this makes sense for now and I hope in the future we can align it even further to avoid confusion 👍